classes/submission/reviewRound/ReviewRoundDAO.inc.php

```diff
@@ -375,6 +376,20 @@ public function _fromRow($row)
 
         return $reviewRound;
     }
+
+    /**
+     * Get assigned reviewers count to a review round by given review round id
+     *
+     * @param  int $id  Review round id for this assigned reviewers count need to determine
+     *
+     * @return int      Number of reviewers assigned to this review round
+     */
+    public function getAssignmentCountByid(int $id): int
+    {
+        return DB::table('review_assignments')
+            ->where('review_round_id', $id)
+            ->count();
+    }
```

Comment:
This should probably go to `ReviewAssignmentsDAO::getCountByReviewRoundId()`. Or if you add the method to get ids suggested above, you could just count the result.

Reply:
Done.